WordPress/src/main/res/layout/my_site_pages_card_footer_link.xml

```diff
@@ -2,19 +2,54 @@
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/my_site_card_footer_link"
-    android:layout_width="wrap_content"
-    android:layout_height="wrap_content">
+    android:id="@+id/my_site_card_footer_link_layout"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:paddingVertical = "@dimen/my_site_dashboard_card_page_create_page_padding_vertical"
```

Comment:
np: Spacing around the "=" sign

Reply:
Fixed with commit [97036b9](https://github.com/wordpress-mobile/WordPress-Android/pull/18322/commits/97036b985beae28b253fd7c99a313fb6eca4bf23)